src/program/lwaftr/setup.lua

```diff
@@ -141,6 +142,9 @@ function lwaftr_app(c, conf)
       prepend(postprocessing_apps_v6, "egress_filterv6")
    end
 
+   -- Add MemoryLog app.
+   config.app(c, "memory_monitor", memory_info.MemoryGauge)
+
```

Comment:
@wingo For a usage example, you mean like this? :-)

I see this as sort of a benefit of it being an app. Super painless configuration. Hack? Maybe.

Reply:
I.e., one-line change for a memory gauge and RRD for every lwaftr process.

Reply:
I'm definitely down with "everybody gets a counter" :-)  I would add this in `lib.ptree` though at least in `lib.ptree.worker`, so that all ptree apps get this counter :)  WDYT?

Reply:
i.e. adding a timer there, as `lib.scheduling` adds an ingress drop monitor.  But if you think that doesn't work, that's ok too.  LMK.

Reply:
Fix in https://github.com/eugeneia/snabb/pull/16 .  wdyt @eugeneia ?

Reply:
Merged, LGTM!